### DISCUSSION TITLE
1306 Library removed from drivers/displays but not in lib nightly build

### DISCUSSION THREAD

**Discussion by JeremyLaurenson** [micropython/micropython#10388] [RP2040 / Pico]
According to the notes the SSD1306 source which used to be in drivers/displays was removed from there as it was moved into linb.

On an RP2 Im not able to import it as per the example:

from ssd1306 import SSD1306_I2C

MPY: soft reboot
Traceback (most recent call last):
  File "boot.py", line 4, in <module>
ImportError: no module named 'ssd1306'
MicroPython v1.19.1-782-g699477d12 on 2022-12-20; Raspberry Pi Pico with RP2040

**Comment by robert-hh** [comment]
You have to copy the file ssd1306.py to the board for being able to import it.

**Reply by JeremyLaurenson** [reply]
Yes, the source of ssd1306.py used to be in the GitHub repo, under drivers/display and was removed.

**Reply by robert-hh** [reply]
Now it is in micropython-lib: https://github.com/micropython/micropython-lib/tree/master/micropython/drivers/display/ssd1306
